### ISSUE TITLE
hyperparameter tuning MERFISH

### ISSUE THREAD

**Issue by YistYU** [WhirlFirst/STEM#5]
Hi thanks for the cool work! Are there some insights on the hyperparameter choices?

I found that the choice sigma and the learning rate varies a lot across different datasets reported in the paper. lr goes from 0.1 to 0.002 and 0.003, and sigma goes from 0.85, to 3 and to 85. Any insights on the hyperparameter selection here, maybe especially on MERFISH data? Basically i found that for the MERFISH dataset that I am trying on, the E_circle loss and E_pred always quickly converge in 50 steps which leads to suboptimal performance. I am struggling on this for a while and would appreciate if there is any suggestion on this case.

Best,

**Comment by WhirlFirst** [comment]
Hi,

We used different parameter settings primarily because the spatial resolutions of the datasets (i.e., the technologies used) vary.

For the sigma parameter, we found empirically that STEM performs best when σ is set to half the mean nearest-neighbor distance of the ST spots. Please refer to Supplementary Figure 11 for more details. Regarding the learning rate, we recommend starting with 0.002, which we used for the MERFISH dataset. You can adjust it based on your own data.

**Comment by YistYU** [comment]
This is super helpful!! Thanks a lot on the prompt response! 

Have a nice day ; )
